app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -45,19 +46,80 @@ import androidx.compose.ui.unit.dp
 import androidx.compose.ui.viewinterop.AndroidView
 import androidx.lifecycle.viewmodel.compose.viewModel
 import ch.hikemate.app.R
+import ch.hikemate.app.model.route.Bounds
+import ch.hikemate.app.model.route.HikeRoute
+import ch.hikemate.app.model.route.LatLong
 import ch.hikemate.app.model.route.ListOfHikeRoutesViewModel
 import org.osmdroid.config.Configuration
 import org.osmdroid.events.MapListener
 import org.osmdroid.events.ScrollEvent
 import org.osmdroid.events.ZoomEvent
 import org.osmdroid.util.GeoPoint
 import org.osmdroid.views.MapView
+import org.osmdroid.views.overlay.Polyline
 
 object MapScreen {
   const val TEST_TAG_MAP = "map"
   const val TEST_TAG_MENU_BUTTON = "menuButton"
   const val TEST_TAG_HIKES_LIST = "hikesList"
   const val TEST_TAG_HIKE_ITEM = "hikeItem"
+
+  const val MIN_HUE = 0
+  const val MAX_HUE = 360
+  const val MIN_SATURATION = 42
+  const val MAX_SATURATION = 98
+  const val MIN_LIGHTNESS = 40
+  const val MAX_LIGHTNESS = 90
+
+  const val STROKE_WIDTH = 10f
+}
+
+/**
+ * Generates a random color in the HSL color space. This function uses the HSL color space to
+ * generate a random color as the HSL is better to generate more visually appealing colors.
+ *
+ * @return The generated color as an [Int].
+ */
```

Comment:
Suggestion: I don't find this documentation very clear. If it wasn't for your IRL explanation, I wouldn't necessarily have gotten it. Maybe mention the fact that HSL has attributes that better represent whether a color will go well on the map or not. For example :

```suggestion
/**
 * Generates a random color in the HSL color space. The function uses HSL to generate a
 * color instead of ARGB because it makes it easier to have a range of colors that we know
 * will render well on the map.
 *
 * @return The generated color as an [Int].
 */
```

Reply:
Love your suggestion! Changed it